windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md

```diff
@@ -4,23 +4,49 @@ description: Warning C28131 The DriverEntry routine should save a copy of the ar
 keywords:
 - warnings listed WDK PREfast for Drivers
 - errors listed WDK PREfast for Drivers
-ms.date: 04/20/2017
-f1_keywords: 
-  - "C28131"
+ms.date: 09/27/2022
+f1_keywords: ["C28131", "NOT_COPYING_NAME", "NOT_COPYING_NAME"]
```

Comment:
"NOT_COPYING_NAME" appears twice in the f1_keywords. Is one of the values missing something? `__WARNING_` perhaps?

Reply:
Done